litestar/connection/base.py

```diff
@@ -301,6 +304,13 @@ def clear_session(self) -> None:
             None.
         """
         self.scope["session"] = Empty
+        self._connection_state.session_id = Empty
+
+    def set_session_id(self, value: str) -> None:
+        self._connection_state.session_id = value
```

Comment:
I don't think this should be added. Users shouldn't be allowed to change the session ID.